include/deal.II/cgal/surface_mesh.h

```diff
@@ -66,6 +78,50 @@ namespace CGALWrappers
     const typename dealii::Triangulation<dim, spacedim>::cell_iterator &cell,
     const dealii::Mapping<dim, spacedim> &                              mapping,
     CGAL::Surface_mesh<CGALPointType> &                                 mesh);
+
+  /**
+   * Performs corefinement and boolean operations on two deal.II cells and put
+   * the result in a CGAL::Surface_mesh object.
```

Comment:
What happens if the intersection is empty?

Reply:
You end up in the default case, with `Assert(res,ExcMessage("The boolean operation you provided doesn't make sense. Please check it."));`

Reply:
Ok. Can you add a comment here? The other option would be to simply return an empty mesh.